src/current/_includes/releases/v23.2/v23.2.9.md

```diff
@@ -0,0 +1,75 @@
+
+<h3 id="v23-2-9-operational-changes">Operational changes</h3>
+
+- `crdb_internal.cluster_execution_insights.txt` and `crdb_internal.cluster_txn_execution_insights.txt` have been removed from the [debug zip]({% link v23.2/cockroach-debug-zip.md %}). These files contained cluster-wide insights for statements and transactions. Users can still rely on the [per-node execution]({% link v23.2/cockroach-debug-zip.md %}#files) insights in `crdb_internal.node_execution_insights.txt` and `crdb_internal.node_txn_execution_insights.txt`. [#125804][#125804]
+- Some debugging-only information about physical plans is no longer collected in the `system.job_info` table for changefeeds, because it has the potential to grow very large. [#126097][#126097]
+- For the [TELEMETRY channel]({% link v23.2/logging.md %}#telemetry), TCL [`sampled_query`]({% link v23.2/eventlog.md %}#sampled_query) events will now be sampled at the rate specified by the setting [`sql.telemetry.query_sampling.max_event_frequency`]({% link v23.2/cluster-settings.md %}#setting-sql-telemetry-query-sampling-max-event-frequency), which is already used to limit the rate of sampling DML statements. [#126728][#126728]
+
+<h3 id="v23-2-9-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug introduced in v23.2.0 in which CockroachDB would hit an internal error when evaluating [`INSERT`s]({% link v23.2/insert.md %}) into [`REGIONAL BY ROW`]({% link v23.2/alter-table.md %}#set-the-table-locality-to-regional-by-row) tables where the source was a `VALUES` clause with a single row and at least one boolean expression. [#125504][#125504] [#126839][#126839]
+- Fixed a bug where a [`DROP ROLE`]({% link v23.2/drop-role.md %}) or [`DROP USER`]({% link v23.2/drop-user.md %}) command could leave references behind inside types, which could prevent subsequent [`SHOW GRANTS`]({% link v23.2/show-grants.md %}) commands from working. [#125806][#125806]
+- Fixed a bug that could lead to descriptors having privileges to roles that no longer exist. Added an automated clean up for [dropped roles]({% link v23.2/drop-role.md %}) inside descriptors. [#125806][#125806]
+- Fixed a bug where a change to a [user-defined type (UDT)]({% link v23.2/create-type.md %}) could cause queries against tables using that type to fail with an error message like: `histogram.go:694: span must be fully contained in the bucket`. The change to the user-defined type could occur either directly from an [`ALTER TYPE`]({% link v23.2/alter-type.md %} statement or indirectly from an [`ALTER DATABASE ... ADD REGION`]({% link v23.2/alter-database.md %}#add-region) or [`ALTER DATABASE ... DROP REGION`]({% link v23.2/alter-database.md %}#drop-region) statement, which implicitly modifies the `crdb_internal_region` UDT. This bug had existed since UDTs were introduced in v20.2. [#125806][#125806]
```

Comment:
```suggestion
- Fixed a bug where a change to a [user-defined type (UDT)]({% link v23.2/create-type.md %}) could cause queries against tables using that type to fail with an error message like: `histogram.go:694: span must be fully contained in the bucket`. The change to the user-defined type could occur either directly from an [`ALTER TYPE`]({% link v23.2/alter-type.md %}) statement or indirectly from an [`ALTER DATABASE ... ADD REGION`]({% link v23.2/alter-database.md %}#add-region) or [`ALTER DATABASE ... DROP REGION`]({% link v23.2/alter-database.md %}#drop-region) statement, which implicitly modifies the `crdb_internal_region` UDT. This bug had existed since UDTs were introduced in v20.2. [#125806][#125806]
```